packages/itwinui-css/src/slider/slider.scss

```diff
@@ -20,7 +21,6 @@ $tick-height: $iui-baseline; // 12px
 
   svg {
```

Comment:
Noticed that icons are black. Previously (currently in main) `iui-icon-default` had colors mixin included. I think we need to include it here to keep icons default color.
`@include iui-icon-color;`

Reply:
Thanks, yes, I thought of leaving it out since Mayank is already handling this in his PR #718 ([Comment](https://github.com/iTwin/iTwinUI/pull/718#discussion_r944672596)). To avoid merging confusion later, maybe I should leave it as it is?